client/my-sites/themes/themes-banner/photo-blog.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+import { localize } from 'i18n-calypso';
+import ThemesBanner from './index';
+
+class PhotoBlogBanner extends PureComponent {
+	render() {
+		const { translate } = this.props;
+		return (
```

Comment:
This is a total matter of taste, so feel free to ignore the suggestion:
I personally like to use functional stateless components in simple cases like this, since they express so nicely the mapping from an input (a set of props) to a result (a component). Are you using the `PureComponent` because it'll easier to add methods to it? Or any other particular reason? (Really just curious 😄)

(Functional stateless equivalent would e.g. be
```js
const PhotoBlogBanner = ( { translate } ) => (
	<ThemesBanner
		themeId="photo-blog"
		/*...*/
	/>
);
```

Reply:
Actually, this approach is so much cleaner and expressive, straight to the point, no fuss...

It's just a simple function returning the component instance. No need to require `PureComponent` or declare extra `render()` functions, etc. Very concise, I love it! :)